keps/sig-node/127-user-namespaces/README.md

```diff
@@ -130,7 +131,7 @@ Here we use UIDs, but the same applies for GIDs.
   the pod (not valid in the host).
 - Benefit from the security hardening that user namespaces provide against some
   of the future unknown runtime and kernel vulnerabilities.
-- Support only stateless pods
+- Initially support stateless and later stateful pods, too.
```

Comment:
The KEP #127 was renamed to "Support User Namespaces in stateless pods". So we can omit this note about stateful Pods and have a separate KEP for those. Or you want to change the scope now?

Reply:
yeah agreed, unless the stateful pieces have already finished (cc @rata @giuseppe )

Reply:
Alright, switched to only stateless pods. But graduation of this feature to GA is then hard to justify.